scripts/gitlab_trigger_client_publish

```diff
@@ -33,16 +34,29 @@ if [ ! -d $WORKSPACE/integration ]; then
   exit 1
 fi
 
+# Get the integration versions to publish
+if [ "$trigger_from_repo" == "meta-mender" ]; then
+  # Special handling on meta-mender: publish master + last two releases
```

Comment:
This won't work for LTS releases (after 2.6, 2.4 won't be released anymore). Might I suggest something based on tag age instead? Something like, the three series whose tags are the newest. Yes, it needs to be three, because of the release we're about to do now, we will release 2.4, 2.5, and 2.6. Even though 2.5 will be the last one, it's hard for the script to know that when we release them at the same time.

Reply:
Good suggestion, I switched to date based tags. However,

> This won't work for LTS releases (after 2.6, 2.4 won't be released anymore).

It would have worked: 2.4.x would have not been updated any more but that is okey as we are not planning more releases there.

> Yes, it needs to be three

Doing it date based yes, it needs to be three (as I implemented it now in the rework)

Reply:
> It would have worked: 2.4.x would have not been updated any more but that is okey as we are not planning more releases there.

Aren't we? 2.4 is an LTS release, valid until July.

Reply:
Of course. Forget that I said, I wasn't thinking right.